Util/trapezoidal_speed_profile.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+
+from RULEngine.robot import Robot, MAX_LINEAR_ACCELERATION, MAX_LINEAR_SPEED
+from Util import Position, Pose
+
+MIN_DISTANCE_TO_REACH_TARGET_SPEED = 0
+
+
+def get_next_velocity(robot: Robot, dt):
+    """Return the next velocity according to a constant acceleration model of a point mass.
+       It try to produce a trapezoidal velocity path with the required cruising and target speed.
+       The target speed is the speed that the robot need to reach at the target point."""
+    next_speed = robot.velocity.position.norm
+    target_speed = robot.path.speeds[1]
+    # cruise_speed = robot.cruise_speed
+    #print(target_speed)
+    #print(next_speed)
+    target_direction = robot.path.points[1] - (robot.pose.position / robot.pose.position.norm)
```

Comment:
I don't really get the math here:
It's the next point in path minus a unitary vector that point in the direction of the robot's position from the  point (0, 0).

Reply:
Wait it's unused